securedrop/db.py

```diff
@@ -264,10 +264,18 @@ class Journalist(Base):
         "JournalistLoginAttempt",
         backref="journalist")
 
+    MIN_USERNAME_LEN = 8
```

Comment:
This is a very long minimum username length. What about 3 characters? That seems more reasonable.